src/main/java/com/bookappstore/model/User.java

```diff
@@ -0,0 +1,32 @@
+package com.bookappstore.model;
+
+import jakarta.persistence.Column;
+import jakarta.persistence.Entity;
+import jakarta.persistence.GeneratedValue;
+import jakarta.persistence.GenerationType;
+import jakarta.persistence.Id;
+import lombok.Getter;
+import lombok.Setter;
+import org.hibernate.annotations.SQLDelete;
+import org.hibernate.annotations.Where;
+
+@Entity
+@Getter
+@Setter
+@SQLDelete(sql = "UPDATE roles SET is_deleted = true WHERE id = ?")
+@Where(clause = "is_deleted = false")
```

Comment:
uppercase true and false, use SQLRestriction instead of a deprecated Where annotation

Reply:
nice tip, thanks